library/core/src/slice/mod.rs

```diff
@@ -1407,6 +1407,25 @@ impl<T> [T] {
         x.slice_contains(self)
     }
 
+    /// Returns `true` if the slice contains an element with the given value.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// # #![feature(contains_ref)]
+    /// let v = [String::from("Hello"), String::from("world")];
+    /// assert!(v.contains_ref("Hello"));
+    /// assert!(!v.contains_ref("Rust"));
+    /// ```
+    #[unstable(feature = "contains_ref", issue = "none")]
```

Comment:
If this gets a "seems plausible" from libs, please do create a tracking issue for this: https://github.com/rust-lang/rust/issues/new?labels=C-tracking-issue&template=tracking_issue.md&title=Tracking+Issue+for+slice::contains_ref

Reply:
I'll remember it !